docs/understanding-tracking-design/json-schema-type-casting-rules/index.md

```diff
@@ -0,0 +1,1384 @@
+
+:::
+
+<table>
+<thead>
+<td>Json Schema</td>
+<td>Databricks Type</td>
+</thead>
+<tbody>
+<tr>
+<td>
+
+  ```json
+{
+    "type": "string",
+    "format": "date-time"
+}
+```
+
+</td>
+<td>
+
+`TIMESTAMP`
+
+</td>
+</tr>
+<tr>
+<td>
+
+  ```json
+{
+    "type": "string",
+    "format": "date"
+}
+```
+
+</td>
+<td>
+
+`DATE`
+
+</td>
+</tr>
+<tr>
+<td>
+
+  ```json
+{
+    "type": "boolean"
+}
+```
+
+</td>
+<td>
+
+`BOOLEAN`
+
+</td>
+</tr>
+<tr>
+<td>
+
+  ```json
+{
+    "type": "string"
+}
+```
+
+</td>
+<td>
+
+`STRING`
+
+</td>
+</tr>
+<tr>
+<td>
+
+  ```json
+{
+    "type": "integer",
+    "minimum": N,
+    "maximum": M
+}
+```
+
+- `M` &le; 2147483647
+- `N` &ge; -2147483648
+
+</td>
+<td>
+
+`INT`
+
+</td>
+</tr>
+<tr>
+<td>
+
+  ```json
+{
+    "type": "integer",
+    "minimum": N,
+    "maximum": M
+}
+```
+
+- `M` &le; 9223372036854775807
+- `N` &ge; -9223372036854775808
+
+</td>
+<td>
+
+`BIGINT`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "integer",
+    "minimum": N,
+    "maximum": M
+}
+```
+
+- `M` &gt;1e38-1
+- `N` &lt;-1e38
+
+</td>
+<td>
+
+`DECIMAL(38,0)`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "integer",
+    "minimum": N,
+    "maximum": M
+}
+```
+
+- `M` &lt; 1e38-1
+- `N` &gt;-1e38
+
+</td>
+<td>
+
+`DOUBLE`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "integer"
+}
+```
+
+</td>
+<td>
+
+`BIGINT`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "number", // OR ["number", "integer"]
+    "minimum": N,
+    "maximum": M,
+    "multipleOf": F
+}
+```
+
+- `M` &le; 2147483647
+- `N` &ge; -2147483648
+- `F` is integer
+
+</td>
+<td>
+
+`INT`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "number", // OR ["number", "integer"]
+    "minimum": N,
+    "maximum": M,
+    "multipleOf": F
+}
+```
+
+- `M` &le; 9223372036854775807
+- `N` &ge; -9223372036854775808
+- `F` is integer
+
+</td>
+<td>
+
+`BIGINT`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "number", // OR ["number", "integer"]
+    "minimum": N,
+    "maximum": M,
+    "multipleOf": F
+}
+```
+
+- `M` &gt; 1e38-1
+- `N` &lt; -1e38
+- `F` is integer
+
+</td>
+<td>
+
+`DECIMAL(38,0)`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "number", // OR ["number", "integer"]
+    "minimum": N,
+    "maximum": M,
+    "multipleOf": F
+}
+```
+
+- `M` &lt; 1e38-1
+- `N` &gt; -1e38
+- `F` is integer
+
+</td>
+<td>
+
+`DOUBLE`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "number", // OR ["number", "integer"]
+    "multipleOf": F
+}
+```
+
+- `F` is integer
+
+</td>
+<td>
+
+`BIGINT`
+
+</td>
+</tr>
+<tr>
+<td>
+
+```json
+{
+    "type": "number", // OR ["number", "integer"]
+    "minimum": N,
+    "maximum": M,
+    "multipleOf": F
+}
+```
+
+</td>
+<td>
+
+`DECIMAL(P,S)`
+
+- `P` &le; 38, where `P` is maximum precision (total number of digits) of `M` and `N`, adjusted for scale (number of digits after the `.`) of `F`.
```

Comment:
I messed up in my previous comments, this one specific line should be on the left, as it distinguishes this row from the next.

The line about P being rounded can stay on the right.